.release/release-metadata.hcl

```diff
@@ -0,0 +1 @@
+url_source_repository = "https://github.com/hashicorp/envconsul"
```

Comment:
The license file link also needs to be added here:

`url_license = "https://github.com/hashicorp/envconsul/blob/main/LICENSE"`

Reply:
This a new thing needed in all the CRT projects? IE. should I add this to consul-template and consul-esm as well?

Reply:
Yes, please! There are currently no failures if you skip the license link, but there will be in the future

Reply:
Added everywhere. Thanks!